src/ClientSink.js

```diff
@@ -0,0 +1,77 @@
+
+const logger = require('./logging');
+const getLastLine = require('./store/file');
+
+const messageTypes = {
+    SequenceNumber: 'sn'
+};
+
+/**
+ * This handles sending the messages to the frontend
+ */
+class ClientSink {
```

Comment:
I was thinking something more generic. Maybe in the future we wan to send other things back to the frontend and this could be the service what is responsible for handling it